backend/support/management/commands/seed_cog_baseline.py

```diff
@@ -71,7 +71,7 @@ def update_cogbaseline_w_csv(filename):
                 & (df["uei"] == cogbaseline_inactive.uei)
             )
         ]
-    save_df_to_cogbaseline(df, "Census")
+    save_df_to_cogbaseline(df, "CENSUS")
```

Comment:
We should remove the second parameter in the function call as we are passing the same value in all calls to this method. The constant can be directly embedded inside the method.